movies-search/src/reducers.jsx

```diff
@@ -1,16 +1,30 @@
 
 import { FETCH_SUCCESS, FETCH_ERROR, SORT_MOVIES } from './actions.jsx';
 import _ from 'lodash';
+import { combineReducers } from 'redux'
 
-export default function reducer(state = [], action) {
+const initialState = {
+    data: [],
```

Comment:
let's define better names for stored properties. you store movies not data. data is to abstract and does not give enough understanding of what is supposed to be stored